### PR TITLE
Add Support FT.ALIASADD FT.ALIASUPDATE FT.ALIASDEL commands

### PR DESCRIPTION
**Pull request by dengliming**

### CODE REVIEW

File: src/test/java/io/redisearch/client/ClientTest.java [hunk 1]
```diff
@@ -992,4 +992,25 @@ public void testConfig() throws Exception {
...
+
+    @Test
+    public void testAlias() throws Exception {
+        Client cl = getClient();
+        cl._conn().flushDB();
+
+        Schema sc = new Schema().addTextField("field1", 1.0);
+        assertTrue(cl.createIndex(sc, Client.IndexOptions.defaultOptions()));
+
+        assertTrue(cl.addAlias("TEST1"));
```
Comment (gkorland) [inline src/test/java/io/redisearch/client/ClientTest.java:1004]: @dengliming can you please add verification after each step that the alias really alias the index?
e.g. by calling info()

Comment (dengliming) [reply src/test/java/io/redisearch/client/ClientTest.java:1004]: That should add a field `alias` to `Query` to support custom alias search. What do you think?

Comment (gkorland) [reply src/test/java/io/redisearch/client/ClientTest.java:1004]:
```java
    @Test
    public void testAlias() throws Exception {
        Client cl = getClient();
        cl._conn().flushDB();

        Schema sc = new Schema().addTextField("field1", 1.0);
        assertTrue(cl.createIndex(sc, Client.IndexOptions.defaultOptions()));
        Map<String, Object> doc = new HashMap<>();
        doc.put("field1", "value");
        assertTrue(cl.addDocument("doc1", doc));


        assertTrue(cl.addAlias("ALIAS1"));
        Client alias1 = getClient("ALIAS1");
        SearchResult res1 = alias1.search(new Query("*").returnFields("field1"));
        assertEquals(1, res1.totalResults);
        assertEquals("value", res1.docs.get(0).get("field1"));
        
        assertTrue(cl.updateAlias("ALIAS2"));
        Client alias2 = getClient("ALIAS2");
        SearchResult res2 = alias2.search(new Query("*").returnFields("field1"));
        assertEquals(1, res2.totalResults);
        assertEquals("value", res2.docs.get(0).get("field1"));

        try {
            cl.deleteAlias("ALIAS3");
            Assert.fail("Should throw JedisDataException");
        } catch (JedisDataException e) {
            // Alias does not exist
        }
        assertTrue(cl.deleteAlias("ALIAS2"));
        try {
          cl.deleteAlias("ALIAS2");
          Assert.fail("Should throw JedisDataException");
      } catch (JedisDataException e) {
          // Alias does not exist
      }
    }
```

File: src/main/java/io/redisearch/Query.java [hunk 2]
```diff
@@ -459,4 +461,13 @@ public Query setSortBy(String field, boolean ascending) {
         _sortAsc = ascending;
         return this;
     }
+
+    public Query indexAlias(String alias) {
```
Comment (gkorland) [inline src/main/java/io/redisearch/Query.java:465]: @dengliming I don't think we need this API, the all idea behind Alias is that application can work with Alias like they work with index and when a new index is created the alias can be switched without effecting the application

File: src/test/java/io/redisearch/client/ClientTest.java (hunk 1 above)
Comment (gkorland) [reply src/test/java/io/redisearch/client/ClientTest.java:1007]: @dengliming I think your original PR is enough just extend your test like the one I added

**Comment by dengliming** [comment]
@gkorland Thanks. All was done.

File: src/main/java/io/redisearch/Query.java (hunk 2 above)
Comment (dengliming) [reply src/main/java/io/redisearch/Query.java]: Revert

**Pull request merged**